daemon/daemon.go

```diff
@@ -237,11 +236,6 @@ func (daemon *Daemon) register(container *Container, updateSuffixarray bool) err
 			cmd := &execdriver.Command{
 				ID: container.ID,
 			}
-			var err error
-			cmd.ProcessConfig.Process, err = os.FindProcess(existingPid)
-			if err != nil {
-				logrus.Debugf("cannot find existing process for %d", existingPid)
-			}
 			daemon.execDriver.Terminate(cmd)
```

Comment:
I think we should check error and log as debug here like we were above... @LK4D4 ?